vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -88,10 +88,16 @@ namespace Microsoft.ReactNative
     )
     void OnPointerEvent(Object view, ReactPointerEventArgs args);
   };
+  
+  [webhosthidden]
+  DOC_STRING(
+    "Enables view managers to track when the view is being "
+    "removed from the visual tree. "
+  )
   interface IViewManagerWithDropViewInstance {
     DOC_STRING(
-        "Enables the view manager to track when the view is "
-        "being removed from the visual tree. "
+        "Method which gets called when the view is being "
+        "removed from the visual tree. "
```

Comment:
can this be more precise? does this happen once the view is removed? after?
by visual tree I'm assuming this means the xaml tree? or is it the shadow tree? if xaml, it would be good to explain how this is different from the XAML `Unloaded` event
also, recommend simplifying to "Invoked when the view has been removed from the visual tree"